operatorconfig/driverconfig/powermax/v2.11.0/node.yaml

```diff
@@ -127,7 +127,7 @@ spec:
               value: "<X_CSI_POWERMAX_ISCSI_ENABLE_CHAP>"
             - name: X_CSI_POWERMAX_PROXY_SERVICE_NAME
               value: "csipowermax-reverseproxy"
-            - name: X_CSI_NODE_CHROOT
+            - name: X_CSI_ISCSI_CHROOT
```

Comment:
Please revert the change, as the new driver 2.11.0 look out for X_CSI_NODE_CHROOT.

Reply:
Reverted the changes.